slicingTool/deprecated/version_topic_comparison.pl

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+our $VERSION = 1.0;
+
+use autodie qw/:all/;
+use Carp;
+use Cwd;
+use File::Slurp;
+use File::Spec::Functions;
+
```

Comment:
Some documentation here would be nice so I know what this script is to be used for.

Reply:
Okay, I'll add some in!

Reply:
Well... you said on the call that this script is already deprecated. If it's going to be deleted soon, I wouldn't bother. My comment was written before I noticed the script was in the "deprecated" directory.